### PR TITLE
2nd attempt at: Merge pull request #2781 from SwiftPackageIndex/dependabot/npm_and_ya…

### PR DESCRIPTION
**Pull request by finestructure**
…rn/dependencies-7337d43281

NPM: Bump the dependencies group with 6 updates

### CODE REVIEW

**Review by daveverwer — APPROVED** [review]

**Comment by finestructure** [comment]
Seems to be working now: https://gitlab.com/finestructure/swiftpackageindex/-/jobs/5759980412

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,6 +21,17 @@ on:
     - cron: "0 8 * * *"
 
 jobs:
+  test-build-front-end:
```
Comment (daveverwer) [inline .github/workflows/ci.yml:24]: Wouldn't failures here be caught in the lint CI step? We can make it more explicit, but I think we are already covered by that step.

Comment (finestructure) [reply .github/workflows/ci.yml:24]: Ah right, that's already over in the other file :+1:

Comment (daveverwer) [reply .github/workflows/ci.yml]: For clarity, we could move the build to here and leave the lint there. It's rather badly named at the moment.

**Pull request merged**